airflow/hooks/dbapi_hook.py

```diff
@@ -70,8 +70,11 @@ def get_uri(self):
         host = conn.host
         if conn.port is not None:
             host += ':{port}'.format(port=conn.port)
-        return '{conn.conn_type}://{login}{host}/{conn.schema}'.format(
+        uri = '{conn.conn_type}://{login}{host}'.format(
```

Comment:
Have you thought to use urrlibunparse here?
https://docs.python.org/3/library/urllib.parse.html 
This could make this code simpler and easier to read.